astro/src/content/quickstarts/quickstart-javascript-express-web.mdx

```diff
@@ -0,0 +1,320 @@
+
+<Aside type="note">
+You can choose to examine the full example of the running code now, or skip below and walk through each step of the process.
+</Aside>
+
+### Full Example
+
+Make sure once again that you are in the directory `complete-application/src`.
+Create a file called `index.ts` with the below code.
+
+`index.ts`
+<RemoteCode url={frontmatter.coderoot + "/complete-application/src/index.ts"}
+  lang="ts" />
+
+#### Run the App!
+You should now be able to start your Express application with 
+
+```shell
+npm run dev
+```
+
+### Walkthrough Example
+
+Make sure once again that you are in the directory `complete-application/src`.
+Create a file called `index.ts` and start by setting up your imports and constants. These constants have already been defined by running the kickstart and `setup.js`.
+
+`index.ts`
+<RemoteCode url={frontmatter.coderoot + "/complete-application/src/index.ts"}
+  lang="ts"
+  tags="top"/>
+
+<Aside type="note">
+If you want to see where the FusionAuth values came from, they can be found in the 
+FusionAuth app (http://localhost:9011/admin). The tenant ID is found on the Tenants
+page. To see the client ID and client secret, go to the Applications page and 
+click the `View` icon under the actions for the Changebank application. You'll 
+find the client id and client secret values in the `OAuth configuration` section.
+</Aside>
+
+#### Static files
+
+We need a way to serve our static files, luckily Express makes this easy with `express.static`. These files will include all of the CSS and images that are required for the UI of the ChangeBank application.
+
+`index.ts`
+<RemoteCode url={frontmatter.coderoot + "/complete-application/src/index.ts"}
+  lang="ts"
+  tags="static"/>
+
+#### Homepage
+
+In order to show off our homepage we need to create a route with the base url `/`. In this route check to see if `userDetails` cookie has any user values and correctly respond with either the `home.html` or `account.html` template. Also if there is no user you can generate the PKCE challenge pair and store them into `userSession` cookie (take note this is an http-only cookie).
+
+<Aside type="note">
+In this file we will use the package [pkce-challenge](https://www.npmjs.com/package/pkce-challenge) to generate a Proof Key for Code Exchange (PKCE) challenge pair. Read more about [PKCE](https://www.oauth.com/oauth2-servers/pkce/authorization-request/).
+</Aside>
+
+`index.ts`
+<RemoteCode url={frontmatter.coderoot + "/complete-application/src/index.ts"}
+  lang="ts"
+  tags="homepage"/>
+
+#### Login Route
+
+In the Changebank application there is a `Login` button, when clicked will open the `/login` route (you can find this in `complete-application/templates/home.html`). Have this redirect them to FusionAuth so that FusionAuth can present them with a login page.
```

Comment:
```suggestion
In the ChangeBank application there is a `Login` button which will open the `/login` route when clicked (you can find this in `complete-application/templates/home.html`). Have this redirect them to FusionAuth so that FusionAuth can present them with a login page.
```

I think that the second sentence is kind of weird